realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -119,6 +119,10 @@ public final boolean isValid() {
         return row != null && row.isAttached();
     }
 
+    protected Table getTable () {
+        return realm.schema.getTable(getClass());
```

Comment:
this will be called only once inside `onCompleted(Long handoverRowPointer)` to create a valid Row